src/tr.i18n.js

```diff
@@ -0,0 +1,541 @@
+module.exports = {
+  'menu.file': {
+    'New Window': 'Yeni Pencere',
+    'File': 'Dosya',
+    'New': 'Yeni',
+    'Open…': 'Aç…',
+    'Save': 'Kaydet',
+    'Save As…': 'Farklı Kaydet…',
+    'Clipboard': 'Pano',
+    'Load SGF': 'SGF Yükle',
+    'Copy SGF': 'SGF Kopyala',
+    'Copy ASCII Diagram': 'ASCII Diyagramı Kopyala',
+    'Game Info': 'Oyun Bilgisi',
+    'Manage Games…': 'Oyunları Yönet…',
+    'Preferences…': 'Tercihler…'
+  },
+  'updater': {
+    'Download Update': 'Güncellemeyi İndir',
+    'View Changelog': 'Değişiklikleri Görüntüle',
+    'Not Now': 'Şimdi Değil',
+    '${appName} v${version} is available now.': '${appName} v${version} şimdi mevcut',
```

Comment:
Interpolated strings use a special syntax. This needs to be 

~~~js
p => `${appName} v${version} şimdi mevcut`
~~~

Please look at the other translation files for more examples.

Reply:
resolved